### PR TITLE
Revert "Azure keyvault trust manager with multiple keystores."

### PR DESCRIPTION
**Pull request by chenrujun**
Reverts Azure/azure-sdk-for-java#20548

### CODE REVIEW

File: sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultTrustManager.java [hunk 1]
```diff
@@ -52,71 +37,31 @@
...
-            } else {
-                addKeyVaultKeystore();
-                addTrustManager(keyStore);
+        this.keyStore = keyStore;
+        if (this.keyStore == null) {
+            try {
+                this.keyStore = KeyStore.getInstance(KeyVaultKeyStore.KEY_STORE_TYPE);
+                this.keyStore.load(null, null);
+            } catch (KeyStoreException | IOException | NoSuchAlgorithmException | CertificateException ex) {
+                ex.printStackTrace();
```
Comment (backwind1233) [inline sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/KeyVaultTrustManager.java:46]: Strong said , we should not use "ex.printStackTrace()"
> we should never have "System.out.println" or "e.printStackTrace()" in our code, use logger instead

**Review by backwind1233 — COMMENTED** [review]
Why revert this

**Comment by yiliuTo** [comment]
> Why revert this

We are now considering about refactor the implemention of supporting a truststore to acquire certs from both kv and jre. The to be reverted design is implemention by trust manager, while we are now thinking about doing that on keystore level directly.

**Comment by chenrujun** [comment]
LGTM.

**Review by yiliuTo — APPROVED** [review]
LGTM

**Pull request merged**